recipes/jbig/all/CMakeLists.txt

```diff
@@ -8,35 +8,27 @@ if(MSVC)
     add_definitions(-D_CRT_SECURE_NO_WARNINGS)
 endif()
 
-add_library(${CMAKE_PROJECT_NAME} source_subfolder/libjbig/jbig.h
-                                  source_subfolder/libjbig/jbig.c
-                                  source_subfolder/libjbig/jbig_tab.c)
-target_link_libraries(${CMAKE_PROJECT_NAME} ${CONAN_LIBS})
-if (WIN32)
-    if (BUILD_SHARED_LIBS)
-        target_compile_definitions(${CMAKE_PROJECT_NAME} PRIVATE _JBIGDLL_=1)
-    endif()
+add_library(${PROJECT_NAME} source_subfolder/libjbig/jbig.c
+                            source_subfolder/libjbig/jbig_tab.c)
+if(MSVC AND BUILD_SHARED_LIBS)
+    target_compile_definitions(${PROJECT_NAME} PUBLIC _JBIGDLL_)
 endif()
 
-include(GNUInstallDirs)
+install(TARGETS ${PROJECT_NAME}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
-set(OTHER_TARGETS)
+install(FILES source_subfolder/libjbig/jbig.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 if (BUILD_EXECUTABLES)
     add_executable(pbmtojbg source_subfolder/pbmtools/pbmtojbg.c)
-    target_link_libraries(pbmtojbg ${CMAKE_PROJECT_NAME})
+    target_link_libraries(pbmtojbg ${PROJECT_NAME})
```

Comment:
I think both `PROJECT_NAME` and `CMAKE_PROJECT_NAME` have the same effect

Reply:
In this context, it's the same value yes. CMAKE_PROJECT_NAME refers to the top level project name, while PROJECT_NAME is the name of the most recent called project().